src/components/notifier/style.css

```diff
@@ -3,18 +3,24 @@
 }
 :host[hidden] {
 	display: none;
+	
 }
 smoothly-trigger {
 	display: inline-block;
 	vertical-align: middle;
-	margin-left: 1.5em;
 }
 aside {
 	text-align: center;
 	vertical-align: middle;
-	width: 100%;
-	height: 2.8em;
-}
+	width: 100%-2em;
+	padding-left: 1em;
+	padding-right: 1em;
+	border-style: solid;
+	border-width: 1px;
+	border-color:196, 160, 0;
```

Comment:
We use variables for the colors rather than ones defined with color codes directly, to make different themes work. E.g. background-color: rgb(var(--smoothly-warning-tint));
If no suitable color variable exists, we should introduce one that fits.